Source/WebCore/dom/FullscreenManager.h

```diff
@@ -113,6 +111,10 @@ class FullscreenManager final : public CanMakeWeakPtr<FullscreenManager>, public
     WTFLogChannel& logChannel() const;
 #endif
 
+    Page* page() const { return document().page(); }
```

Comment:
These can stay public. They do have use cases, even in our smart pointer world. When calling a trivial getter, using a raw pointer overload is more efficient and safe, e.g.
```
page()->protectedElement()->nonTrivialFunction();
```

if `protectedElement()` is a trivial getter, there is no reason to protect the page here, protecting the Element is enough.